planetary-computer/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "planetary-computer" %}
+{% set version = "0.1.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/planetary-computer-{{ version }}.tar.gz
+  sha256: a451b9dfd266d077e7a64a164b73d761f6a1d8bc5b63f413f8ceb35b33507a71
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - click >=7.1
+    - pydantic >=1.7.3
+    - pystac <0.6,>=0.5.6
+    - python
+    - pytz >=2020.5
+    - requests >=2.25.1
+
+test:
+  imports:
+    - planetary_computer
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://pypi.org/project/planetary-computer/
+  summary: Planetary Computer SDK for Python
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - TomAugspurger
+    - lossyrob
```

Comment:
@lossyrob when you get a chance can you +1 to indicate that you're OK with being a maintainer here?

Reply:
@TomAugspurger good by me!